client/src/App.js

```diff
@@ -126,8 +134,35 @@ export default class App extends Component {
     window.location.reload();
   };
 
+  updateTheme = (event) => {
+    const { value } = event.target;
+    localStorage.setItem(`theme-${this.state.user}`, value);
+    this.setState({ theme: value });
+  };
+
   render() {
-    const { user, isUpdateAvailable, installPromptEvent, updateRequired } = this.state;
+    const { user, isUpdateAvailable, installPromptEvent, updateRequired, theme } = this.state;
+
+    // add theme
+    if (user) {
```

Comment:
I believe we can keep this activated even without a user logged in (it will follow `automatic` if we set it as default)